### PR TITLE
Adds the universal scanner to lathes

### PR DESCRIPTION
**Pull request by jlsnow301**
## About The Pull Request
Adds the basic tool back to autolathes and cargo protolathe
This was in the game before #70314 so I think it's a fix more than an addition, but whatever
## Why It's Good For The Game
Sometimes I like to sell things from custom vendors and I don't want to have to hunt down a roundstart cargonian just to get a price tagger.
## Changelog
:cl:
add: You can print universal scanners (export scanners, price taggers, sales taggers) from the lathe again
/:cl:

### CODE REVIEW

File: code/modules/research/designs/autolathe/multi-department_designs.dm [hunk 1]
```diff
@@ -537,3 +537,16 @@
 		RND_CATEGORY_MODULAR_COMPUTERS + RND_SUBCATEGORY_MODULAR_COMPUTERS_FRAMES,
 	)
 	departmental_flags = DEPARTMENT_BITFLAG_SCIENCE | DEPARTMENT_BITFLAG_ENGINEERING
+
+/datum/design/universal_scanner
+	name = "Universal Scanner"
+	desc = "A utility scanner that fills multiple roles: Exports, sales, vendor pricetags."
```
Comment (ArcaneMusic) [inline code/modules/research/designs/autolathe/multi-department_designs.dm:543]:
```suggestion
	desc = "A utility scanner that fills multiple roles: exports, sales, and vendor price tags."
```

**Review by ArcaneMusic — COMMENTED** [review]
For the best, oops on my part

**Comment by moocowswag** [comment]
Finally, having to buy these damn things from public tool storage vendor (which mostly sells items you can print from autolathe) was so frustrating cause alot of jobs could barely afford it.

It was a pretty big balance nerf slapped inside of a "QOL" chance.

File: code/modules/research/designs/autolathe/multi-department_designs.dm (hunk 1 above)
Comment (jlsnow301) [reply code/modules/research/designs/autolathe/multi-department_designs.dm:543]: (I cringe at myself saying this)
Exports would still be capitalized

**Review by ArcaneMusic — APPROVED** [review]

**Pull request merged**